components/03-modules/dashboard/items/items.hbs

```diff
@@ -1,23 +1,29 @@
 <div class="dashboard-items {{ class }}">
     {{#if title }}
-    <h3 class="heading heading--fifth-level dashboard-items__title {{ titleClass }}">
-        {{ title }}
-    </h3>
+        <h3
+            class="dashboard-items__title {{ titleClass }}">
```

Comment:
No need for breaking a line here, it can be in one line

Reply:
changed